apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryBigNumeric.cs

```diff
@@ -331,15 +188,15 @@ public override string ToString()
         /// <param name="value">The integer value to convert.</param>
         /// <returns>The result of the conversion.</returns>
         public static implicit operator BigQueryBigNumeric(long value) =>
-            new BigQueryBigNumeric(s_integerScalingFactor * value);
+            new BigQueryBigNumeric(s_numericType.ValueFromLong(value));
```

Comment:
The method names should be ValueFromInt64 and ValueFromUInt64 IMO. It doesn't matter *much* for internal methods, but I'd personally prefer to see the CLR type names.